src/Components/Main.js

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from "react";
+import firebase from "./firebase";
+// import firebase modules
+import { getDatabase, onValue, push, ref, remove } from "firebase/database";
+import { getAuth } from "firebase/auth";
+
+function Main() {
+  // create country state that will store database info to Firebase
+  const [entries, setEntries] = useState([]);
+  // create stateful value that is bound to input
+  const [userInput, setUserInput] = useState("");
+  // create login state
+  const [userlogin, setUserLogin] = useState(false);
+
+  // event that will fire every time there is a change in our input
+  const handleInputChange = (event) => {
+    setUserInput(event.target.value);
```

Comment:
just noticing, this is only one line of code, it might be better for it to go directly in the props object.